### ISSUE TITLE
bug: 'no such file or directory as overflow.wasm'

### ISSUE THREAD

**Issue by Bobakanoosh** [chearon/dropflow#12]
I'm not able to get dropflow running with the basic example from the README.

Reproduction Link: https://github.com/Bobakanoosh/dropflow-not-working

Reproduction Steps:
1. `pnpm i`
2. `pnpx tsx main.ts`

Yields:
```
Error: ENOENT: no such file or directory, open 'C:\Users\jackb\Documents\Development\testing\dropflow\node_modules\.pnpm\dropflow@0.1.0\node_modules\dropflow\dist\overflow.wasm'
```

**Comment by chearon** [comment]
Sorry about that, it looks like I accidentally published with an outdated build. I just released v0.1.1 which won't throw that error. But I also had some things wrong in the readme 🤦‍♂️ [updated code is here](https://github.com/chearon/dropflow?tab=readme-ov-file#usage) which makes your code run if you supply it a path to a TTF/OTF.

**Comment by Bobakanoosh** [comment]
Working now! thanks
